engine/src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -323,9 +323,10 @@ trait CromwellApiService extends HttpInstrumentation {
     }
   }
 
+  private lazy val metadataBuilderRegulatorActor = actorRefFactory.actorOf(MetadataBuilderRegulatorActor.props(serviceRegistryActor))
```

Comment:
Would it make sense to push the regulator behind the service registry rather than having it awkward inserted in front?